amundsen_application/api/utils/notification_utils.py

```diff
@@ -21,14 +23,15 @@ def send_notification(*, notification_type: str, options: Dict, recipients: List
     # TODO: write tests
     try:
         mail_client = get_mail_client()
-
         notification_content = get_notification_content(
             notification_type=notification_type,
-            options=options
+            options=options,
+            recipients=recipients,
+            sender=sender
         )
 
-        if sender in recipients:
-            recipients.remove(sender)
+        # if sender in recipients:
```

Comment:
You can still leave this in, for testing purposes if you want to make sure you get the email so can do that in our custom client. I prefer that option so that we can still check that the email list is what it should be and that all logic is working correctly. And then just append yourself to the list for testing purposes, or temporarily replace the recipients list with yourself.